solidity/script/avs/eigenlayer_addresses.json

```diff
@@ -0,0 +1,42 @@
+{
+  "ethereum": {
+    "proxyAdmin": "0x75EE15Ee1B4A75Fa3e2fDF5DF3253c25599cc659",
+    "delegationManager": "0x39053D51B77DC0d36036Fc1fCc8Cb819df8Ef37A",
+    "avsDirectory": "0x135DDa560e946695d6f155dACaFC6f1F25C1F5AF",
+    "paymentCoordinator": "",
+    "strategies": [
+      {
+        "name": "cbETH",
+        "strategy": "0x54945180dB7943c0ed0FEE7EdaB2Bd24620256bc"
+      },
+      {
+        "name": "stETH",
+        "strategy": "0x93c4b944D05dfe6df7645A86cd2206016c51564D"
+      },
+      {
+        "name": "Beacon Chain ETH",
+        "strategy": "0xbeaC0eeEeeeeEEeEeEEEEeeEEeEeeeEeeEEBEaC0"
+      }
+    ]
+  },
+  "holesky": {
+    "proxyAdmin": "",
```

Comment:
afaict the current deploy script would set the proxy admin to the empty address?

Reply:
yup